UpdatePublishedVersions.ps1

```diff
@@ -65,6 +65,8 @@ function Update-GitHub-File(
 
     message "Getting the `"sha`" of the current contents of file '$owner/$repo/$path'"
 
+    [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bxor [Net.SecurityProtocolType]::Tls12
```

Comment:
Seeing `-bxor` here I realize it's a bad idea--could disable it if it's already defined! Should be `-bor`.